src/TCommandLine.cpp

```diff
@@ -1288,3 +1293,94 @@ void TCommandLine::clearSuggestions()
 {
     commandLineSuggestions.clear();
 }
+
+void TCommandLine::slot_adjustAccessibleNames()
+{
+    bool isMultipleProfilesActive = (mudlet::self()->getHostManager().getHostCount() > 1);
+    const QString hostName{mpHost ? mpHost->getName() : QString()};
+    switch (mType) {
+    case MainCommandLine:
+        if (isMultipleProfilesActive) {
+            setAccessibleName(tr("Input line for \"%1\" profile.",
+                                 // Intentional comment to separate arguments
+                                 "Item name for acccessibilty system. Phrased so that a changable possive suffix "
+                                 "can be avoided but kept as short as possible for this, the main input command "
+                                 "line when more than one profile is loaded.").arg(hostName));
+            setAccessibleDescription(tr("Type in text to send to the game server for the \"%1\" profile, or enter an alias "
+                                        "to run commands locally.",
+                                        // Intentional comment to separate arguments
+                                        "Item description for acccessibilty system. Phrased so that a changable possive "
+                                        "suffix can be avoided. Used when more than one profile is loaded.").arg(hostName));
+        } else {
+            setAccessibleName(tr("Input line.",
+                                 // Intentional comment to separate arguments
+                                 "Item name for acccessibilty system. To be kept as short as possible for this, "
```

Comment:
"To be kept as short as possible for this, the main input command line when only one profile is loaded." also reads rather awkward, same goes for ones below.